src/index.html

```diff
@@ -22,6 +22,359 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo of Moyo"
+        />
+      </a>
+      <nav class="navbar">
+        <ul class="navbar__ul">
+          <li class="navbar__li">
+            <a
+              href="#"
+              class="is-active navbar__link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="navbar__li">
```

Comment:
change it in all the cases
https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules
<img width="1061" alt="Знімок екрана 2024-08-13 о 17 32 59" src="https://github.com/user-attachments/assets/2f01443f-97cc-4118-9615-cd5d3601eff0">

```suggestion
      </a>
      
      <nav class="navbar">
        <ul class="navbar__ul">
          <li class="navbar__li">
            <a
              href="#"
              class="is-active navbar__link"
            >
              Apple
            </a>
          </li>
          
          <li class="navbar__li">
```